app/webpacker/components/wca/FormBuilder/input/MarkdownEditor.js

```diff
@@ -113,16 +121,17 @@ function getOptions(disabled) {
         onError(e);
       }
     },
-    disabled,
+    ...imageOptions,
   };
 }
 
 export default function MarkdownEditor({
   id,
   value,
   onChange,
+  editorOptions = DEFAULT_EDITOR_OPTIONS,
```

Comment:
Not sure whether such a complex configuration object is needed right now. We don't use this editor a lot to be honest, so you might be trying to future-proof a use-case that will actually never happen (for reference, I think we never ever touched it before this PR of yours)

Anyhow, this thing alone won't block your PR, so you're free to keep it if you feel strongly about this idea. My other remarks are more important

Reply:
Actually, on second thought here's an argument against your configuration hash idea (sorry!):
If you pass a complex configuration hash, the user is responsible for just "knowing" how the options are called and doing research on which options to pass.
If you have a property `imageUploadEnabled`, it's very clear from the name alone what that does, and only one developer (right now, you :smile: ) is responsible for mapping this boolean property to the correct internal configuration settings. Once that mapping is done (and the library doesn't introduce breaking changes), everybody else can use it "for free"

Reply:
Okay done.